content/sensu-enterprise/3.1/changelog.md

```diff
@@ -45,6 +46,29 @@ _NOTE: Sensu Enterprise is built on Sensu Core. Sensu Core changes are documente
 - [Enterprise 1.14.0 Release Notes](#enterprise-v1-14-0)
 - [Enterprise 1.13.0 Release Notes](#enterprise-v1-13-0)
 
+## Enterprise 3.1.0 Release Notes {#enterprise-v3-1-0}
+
+**July 30, 2018** &mdash; Sensu Enterprise version 3.1.0 has been
+released and is available for immediate download. Please note the
+following improvements:
+
+### CHANGES {#enterprise-v3-1-0-changes}
+
+- **IMPROVEMENT**: The EC2 integration now supports cross-account access,
+allowing you to use IAM-defined trust relationships to access a Sensu Enterprise
+instance from EC2 clients across multiple AWS accounts. See the
+[EC2 integration docs](../../integrations/ec2#cross-account-access)
+to learn about setting up cross-account access.
+
+- **IMPROVEMENT**: The PagerDuty integration now supports PagerDuty Events API v2.
+See the [PagerDuty integration docs](../../integrations/pagerduty) to add a v2 API key to Sensu Enterprise.
+No changes are required for existing v1 API integrations using a `service-key`.
```

Comment:
Maybe use the word attribute to call out that `service-key` is a Sensu attribute?

Reply:
@apaskulin I think it should be `service_key`